### DISCUSSION TITLE
Ability to open beatmaps to an external browser.

### DISCUSSION THREAD

**Discussion by s1iqbal** [ppy/osu#16636] [Ideas] [answered]
It could be as simple as holding a hotkey and clicking the map.

**Comment by bdach** [comment, marked as answer]
this is already possible via the beatmap overlay. click the icon next to the beatmap title.

however the eventual goal is functional parity of the overlay with web, so that anything that can be done from web can also be done from client.

**Reply by s1iqbal** [reply]
Had no idea this existed, thank you!

**Comment by ItsJustSomeDude** [comment]
It can be done currently by Right-clicking, going to the More details option to get to the beatmap overlay, then clicking the arrow by the name.

I can't speak for a hotkey or anything being added.
